OpenAPI/OpenAPI.NET/references-openapi.md

```diff
@@ -0,0 +1,665 @@
+    version: 1.0.0
+paths:
+    /person/{id}:
+        get:
+          responses:
+            200:
+              description: ok
+              content:
+                application/json:
+                  schema:
+                    $ref: 'OAS-schemas.yaml#address'
+```
+
+```yaml
+# OAS-schemas.yaml file
+openapi: 3.1.0
+info:
+    title: OpenAPI document containing reusable components
+    version: 1.0.0
+components:
+  schemas:
+      person:
+        type: object
+        properties:
+            name:
+                type: string
+            address:
+                $anchor: address
+                type: object
+                properties:
+                    street:
+                        type: string
+                    city:
+                        type: string
+
+```
+
+
+#### An external fragment [Not supported]
+
+```yaml
+openapi: 3.1.0
+info:
+    title: Reference to an external OpenApi document component
+    version: 1.0.0
+paths:
+    /person/{id}:
+        get:
+          responses:
+            200:
+              description: ok
+              content:
+                application/json:
+                  schema:
+                    $ref: 'OAS-schemas.yaml#/person/properties/address'
+```
+
+```yaml
+# OAS-schemas.yaml file
+person:
+  type: object
+  properties:
+    name:
+        type: string
+    address:
+        type: object
+        properties:
+            street:
+                type: string
+            city:
+                type: string
+```
+
+
+### What can a JSON Schema identifier reference target
+
+- Reference an internal component using a $id
+- Reference an internal subschema using a $id
+- Reference external components and subschemas using a $id
+
+#### Reference an internal component using a $id
+```yaml
+openapi: 3.1.0
+info:
+    title: Reference an internal component using id
+    version: 1.0.0
+paths:
+    /person/{id}:
+        get:
+          responses:
+            200:
+              description: ok
+              content:
+                application/json:
+                  schema:
+                    $ref: 'https://schemas.acme.org/person'
+components:
+      person:
+        $id: 'https://schemas.acme.org/person'
+        type: object
+        properties:
+            name:
+                type: string
+            address:
+                type: object
+                properties:
+                    street:
+                        type: string
+                    city:
+                        type: string
+```
+
+#### Reference an internal subschema using a $id  [Not supported yet]
+
+```yaml
+openapi: 3.1.0
+info:
+    title: Reference an internal subschema using id
+    version: 1.0.0
+
+paths:
+    /person/{id}/address:
+        get:
+          responses:
+            200:
+              description: ok
+              content:
+                application/json:
+                  schema:
+                    $ref: 'https://schemas.acme.org/address'
```

Comment:
I'm assuming this is based on URI conventions. If the id was /address, it'd have been equivalent. But if it were `./address` the ref should have been `https://schema.acme.org/person/address` ? 

Maybe worth calling out even if we don't implement it/fully spec it yet.